src/lumigo_opentelemetry/libs/json_utils.py

```diff
@@ -211,3 +217,12 @@ def safe_convert_bytes_to_string(value: Any) -> Any:
         return value.decode("utf-8")
     except Exception:
         return value
+
+
+def _get_max_size():
```

Comment:
This is repeated logic with what is added to __init__.py. Can we reuse it in the __init__ as well?